src/sentry/lang/native/utils.py

```diff
@@ -25,6 +25,7 @@
 # Mapping of well-known minidump OS constants to our internal names
 MINIDUMP_OS_TYPES = {
     'Mac OS X': 'macOS',
+    'Windows NT': 'Windows',
```

Comment:
Do we want to go with Windows NT? Why not just Windows?

Reply:
That's exactly what this line does (see screenshot above). Breakpad reports "Windows NT", so I map it back to "Windows".

Reply:
oh 😓